### DISCUSSION TITLE
Uninstall FileBrowser

### DISCUSSION THREAD

**Discussion by BatterBits** [tteck/Proxmox#848] [Help Needed] [answered]
Hi,
I installed in the wrong place, can you advise how to uninstall so that I can re-run your fabulous script to install it in the correct location?

I tried disabling the service and deleting the DB but i errors when i try to re-install....

**Comment by tteck** [comment, marked as answer]
https://github.com/tteck/Proxmox/issues/688

**Reply by BatterBits** [reply]
Thanks so much tteck - and thanks for all your great scripts. Sorry I searched for "uninstall" and didn't find this answer.
